beaconApp/src/main/java/com/pulselink/beacon/ui/ThreadScreen.kt

```
@@ -492,7 +503,12 @@ fun TimePickerDialog(
 
 @OptIn(ExperimentalFoundationApi::class)
 @Composable
-private fun LazyItemScope.MessageBubble(message: SmsMessageItem, reactions: List<Reaction> = emptyList(), theme: ThemePalette) {
+private fun LazyItemScope.MessageBubble(
```

detekt (1.23.8)

[warning] The function MessageBubble appears to be too complex based on Cognitive Complexity (complexity: 57). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CognitiveComplexMethod)